src/psyclone/tests/domain/gocean/transformations/gocean_opencl_trans_test.py

```diff
@@ -109,6 +109,9 @@ def test_transformation_name():
 def test_validate_unsupported_api():
     ''' Check that attempting to apply an OpenCL transformation to a Dynamo
     InvokeSchedule raises the expected error. '''
+    # Undo the effect of the 'auto-use' fixture (at the top of this file)
+    # that sets-up the Config object for the GOcean API.
+    Config._instance = None
```

Comment:
It's because the fixture is automatically used for every test and that means the Config object will have the API set to GOcean. By setting this to None we force a new Config object to be created. I've extended the comment to try to make this clearer.